sites/intellij-guide/contents/tips/find-usages/index.md

```diff
@@ -0,0 +1,21 @@
+---
+date: 2020-10-05
+title: 'Find Usages'
+technologies: []
+topics: [navigation]
+author: hs
+subtitle: 'Use Find Usages to search for references in your code base.'
+thumbnail: ./thumbnail.png
+cardThumbnail: ./card.png
```

Comment:
Thanks for adding the card. Later, when you're hooked up to Twitter, it will be worth it.